GVFS/GVFS.Service/Handlers/GetActiveRepoListHandler.cs

```diff
@@ -69,23 +69,31 @@ public void Run()
         private bool IsValidRepo(string repoRoot)
         {
             string gitBinPath = GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath();
-            string hooksPath = ProcessHelper.WhereDirectory(GVFSPlatform.Instance.Constants.GVFSHooksExecutableName);
-            GVFSEnlistment enlistment = null;
+            string hooksPath = null;
 
-            try
+            string hooksVersion = null;
+            string error = null;
+            if (GVFSPlatform.Instance.TryGetGVFSHooksPathAndVersion(out hooksPath, out hooksVersion, out error))

```

Comment:
It looks like we're silently ignoring failures of `TryGetGVFSHooksPathAndVersion`.

At a minimum we should log the error if `TryGetGVFSHooksPathAndVersion` fails.